IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_PlantCommon/DocPropertySet.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_PlantCommon" UniqueId="bff7eedc-8844-446a-b6c2-fd5bdaad0968" ApplicableType="IfcPlant">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_PlantCommon" UniqueId="bff7eedc-8844-446a-b6c2-fd5bdaad0968" ApplicableType="IfcPlant" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
PSET_OCCURRENCEDRIVEN?

Reply:
uumm not sure, i know it doesnt have a type yet but..

Reply:
Should this not be the same as with other entities that do not have a type?